pilot/pkg/networking/core/route/retry/retry.go

```diff
@@ -131,6 +131,12 @@ func ConvertPolicy(in *networking.HTTPRetry, hashPolicy bool) *route.RetryPolicy
 		}
 	}
 
+	if in.Backoff != nil {
+		out.RetryBackOff = &route.RetryPolicy_RetryBackOff{
+			BaseInterval: in.Backoff,
```

Comment:
Do we need some rules in validation.go? Like non-negative duration, etc

Reply:
good point added